From 18d393de8a1927562605e6a9aa60502078a0cfb9 Mon Sep 17 00:00:00 2001
From: Lijun Fang <fanglijun3@huawei.com>
Date: Fri, 3 Dec 2021 18:15:39 +0800
Subject: [PATCH] svm: Enable CONFIG_HISI_SVM by default
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 18d393de8a1927562605e6a9aa60502078a0cfb9


ascend inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I4JMM0
CVE: NA

--------

Enable CONFIG_HISI_SVM by default.

Signed-off-by: Lijun Fang <fanglijun3@huawei.com>
Reviewed-by: Weilong Chen <chenweilong@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/arm64/configs/openeuler_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/openeuler_defconfig b/arch/arm64/configs/openeuler_defconfig
index f23a30a6fb01..d94a72fd0ffe 100644
--- a/arch/arm64/configs/openeuler_defconfig
+++ b/arch/arm64/configs/openeuler_defconfig
@@ -3296,6 +3296,7 @@ CONFIG_TCG_TIS_ST33ZP24_I2C=y
 CONFIG_TCG_TIS_ST33ZP24_SPI=y
 # CONFIG_XILLYBUS is not set
 CONFIG_PIN_MEMORY_DEV=m
+CONFIG_HISI_SVM=y
 # end of Character devices
 
 # CONFIG_RANDOM_TRUST_CPU is not set
-- 
2.26.2

